cni/network/plugin/main.go

```diff
@@ -149,6 +149,11 @@ func rootExecute() error {
 	)
 
 	config.Version = version
+
+	// based on the args, parse the stateless flag
+	stateless := common.GetArg(common.OptStatelessCNI).(bool)
```

Comment:
this won't work..cni is invoked by CRI and can't change cri to invoke cni with this arg.. Lets create separate directory for stateless cni as discussed before